js/apps/admin-ui/src/utils/useFetch.ts

```diff
@@ -21,12 +21,11 @@ export function useFetch<T>(
   callback: (param: T) => void,
   deps?: DependencyList,
 ) {
-  const onError = useErrorHandler();
+  const { showBoundary } = useErrorBoundary<Error>();
```

Comment:
Although the `Error` generic here is technically more correct, we actually do not know the type of `error` in `catch()`, so this can be omitted.
```suggestion
  const { showBoundary } = useErrorBoundary();
```